keda/templates/webhooks/validatingconfiguration.yaml

```diff
@@ -26,6 +26,7 @@ webhooks:
       name: {{ .Values.webhooks.name }}
       namespace: {{ .Release.Namespace }}
       path: /validate-keda-sh-v1alpha1-scaledobject
+      port: {{ .Values.webhooks.port | default 9443 }}
```

Comment:
This doesn't work because the port defined here is the service port (443) and the chart doesn't allow changing it. If you want to modify it, please add another parameter and update the service port as well.
Before the change, it worked because the default value is 443

Reply:
Just to clarify, this is the port used in the control plane to reach the hook (the service port), not the hook pod port itself (the target port)